src/Stack/Options/CleanParser.hs

```diff
@@ -19,4 +19,6 @@ cleanOptsParser = CleanShallow <$> packages <|> doFullClean
         flag'
             CleanFull
             (long "full" <>
-             help "Delete all work directories (.stack-work by default) in the project")
+             help "Delete the local stack working directory (.stack-work by default).")
```

Comment:
Ah — this is a subtle change in meaning (probably my fault). It is possible for a project to have multiple .stack-work directories. The old description indicates all are deleted; this implies only one is deleted.

Reply:
Oh my bad, I wasn't aware that you could have multiple .stack-work directories. Ill fix that

Reply:
I'm not sure how stack actually behaves, mind.